examples/src/index.js

```diff
@@ -101,15 +101,17 @@ window.onload = () => {
 
     const wordsDiv = document.getElementById('transcript-list')
     wordsDiv.innerHTML = words
-      .map(word => (word.isFinal ? `<li><b>${word.value}</b></li>` : `<li>${word.value}</li>`))
+      .map(word =>
+        word.isFinal ? `<li><b>${word.value} [${word.index}]</b></li>` : `<li>${word.value} [${word.index}]</li>`
+      )
       .join('')
   }
 
   function updateEntities(entities) {
     const entitiesDiv = document.getElementById('entities-list')
     entitiesDiv.innerHTML = entities
       .map(entity => {
-        const t = `${entity.type} - ${entity.value} [${entity.startPosition} - ${entity.endPosition}]`
+        const t = `${entity.type} - ${entity.value} [${entity.startPosition} - ${entity.endPosition})`
```

Comment:
Is this a typo?

Reply:
No, this is intentional, to indicate that the interval is half-open from the right (i.e. `endPosition` is not included).